app/assets/javascripts/global_listeners.js

```diff
@@ -30,7 +30,6 @@ $(function() {
     // Show chilly bin if it's hidden
     $('.hide_chilly_bin.pressed').click();
     var newStoryElement = $('#chilly_bin div.story:last');
-    $('#chilly_bin').scrollTo(newStoryElement, 100);
```

Comment:
Aren't we gonna scroll to the element anymore? Just highlighting it?

Reply:
Yes, we aren't creating the story in the last position of chilly bin anymore, we moved to the first one, but this was a missing change, a bug remained :/

Reply:
Oh, I see! Maybe we could have it back using event listeners (like `projectView.on('addStory', function(newStoryElement) { ... });` or something like that), something that doesn't assume the position of the new story. Do we have a ticket for adding it again?

Reply:
Well, I think we don't need scroll anymore, since it's created in the top, as the first story of the column. Highlight is only a fancy thing we are doing to show this story is created there.

Reply:
Makes sense!